index.html

```diff
@@ -14,11 +14,73 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
 </head>
-<body>
-  <header>
+<body class="background-cream">
+  <header class="background-purple">
+    <div class="row container" id="header">
```

Comment:
No element should be both a `container` and a `row`. You should give this `container` a child of a `row`